var/spack/repos/builtin/packages/libtheora/package.py

```diff
@@ -40,7 +67,63 @@ def autoreconf(self, spec, prefix):
         else:
             sh("./autogen.sh", "prefix={0}".format(prefix))
 
-    def configure_args(self):
-        args = []
-        args += self.enable_or_disable("doc")
-        return args
+
+class MSBuildBuilder(MSBuildBuilder):
+    def is_64bit(self):
+        return "64" in self.pkg.spec.target.family
```

Comment:
@scheibelp I've refactored this the same way it was done for XZ. That seem right to you as well?

Reply:
Yes